merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/GetActivityAction.java

```diff
@@ -41,17 +42,17 @@ public GetActivityAction(final Connection connection) throws SQLException {
     this.statement = connection.prepareStatement(sql);
   }
 
-  public ActivityInstance get(final long planId, final long activityId)
+  public ActivityInstance get(final long planId, final ActivityInstanceId activityId)
```

Comment:
IntelliJ is telling me `GetActivityAction` is unused - is there a reason to keep it around, or can we delete it?